devel/cdecl/Portfile

```diff
@@ -14,19 +15,31 @@ description         Composing and deciphering C (or C++) type declarations \
                     or casts.
 long_description    cdecl (see-deh-kull) is a program for composing \
                     and deciphering C (or C++) type declarations or casts, \
-                    a.k.a. \"gibberish.\" It can be used interactively on a \
-                    terminal or accept input from either the command line \
+                    a.k.a. “gibberish”. It can be used interactively \
+                    on a terminal or accept input from either the command line \
                     or standard input.
 
-checksums           rmd160  f3b89d81fc801158fa4eca7876b6bf40789232ff \
-                    sha256  53f60863e8cb597234ff08f619b47f7d7e11fc8d83543821b0e88acf11a714e4 \
-                    size    299046
+checksums           rmd160  5536a94ead950676bd2ed2c7769ea92c8f22a8af \
+                    sha256  ba17cde63b42e772bcf7b8a67bc8979eb9df49b7c0299ea35d885eac7d4a1cd2 \
+                    size    779556
 
 depends_build-append    port:autoconf \
                         port:automake
 
 depends_lib-append      port:ncurses \
-                        port:readline
+                        port:readline \
+                        port:flex \
+                        port:bison
```

Comment:
aren't "flex" and "bison" only build dependencies?

Reply:
Bison and Flex are needed only if you build from _scratch_ using the code _only_ in the repo.  Released tarballs, however, already have the generated `lexer.c` and `parser.c` built, so Bison and Flex are _not_ required to build from a tarball.

Reply:
> Bison and Flex are needed only if you build from scratch using only the code in the repo. Released tarballs, however, already have the generated `lexer.c` and `parser.c` built, so Bison and Flex are _not_ required to build from a tarball.

@paul-j-lucas @reneeotten Thank you! Will remove these then.